.license-template

```diff
@@ -0,0 +1,17 @@
+*
```

Comment:
yes this needs to not have space before it, while all the others need it and yes we need the `*`

Reply:
Ah, nice catch. I tried to setup go-header a few months ago and it kept failing for some reason, but this was probably it.